### DISCUSSION TITLE
ERROR: ERROR: java.lang.IllegalStateException: No matching configuration files found for

### DISCUSSION THREAD

**Discussion by emoxam** [kubesphere/community#417] [Q&A]
Hello! I try to deploy deployment to k8s cluster.
But when i start my pipeline i get an error
Starting Kubernetes deployment
ERROR: ERROR: java.lang.IllegalStateException: No matching configuration files found for **/deployment.yaml

Where should i put my yaml to ?
Thanks

**Comment by sitaorg2022** [comment]
you have solution?

**Reply by emoxam** [reply]
> you have solution?

I guess thats what i used **/*.yaml
P.S. BTW i then switched to devtron because kubesphere uses the same legacy jenkins plugin that does not work but if you'll find a newwet vesrion it wil not support ingresses, so i need to make k apply -f by hands, why not to use this command and pipeeline? thats where i stopped (devtron is a reserved varian for me)
